### PR TITLE
Fixed issue #09876: Different bogus with thousand seperator

### PR DESCRIPTION
**Pull request by Shnoulle**
Dev: Fixed : can not have thousand and no thousand in same page
Dev: Allow some fix/option in template
Dev: Allow fix and option in plugin
Dev: Allow empty, but disallow 0 : option when send the function (user without XSS can call twice)

### CODE REVIEW

**Comment by Shnoulle** [comment]
No PHP fix needed : js do the job.

File: scripts/numerical_input.js [hunk 1]
```diff
@@ -1,54 +1,96 @@
-$(document).ready(function () {
-    if (typeof LEMradix === 'undefined') { return; }
 
-    if (LEMradix == ',')
+/**
+ * replace the LEMval function, do it at ready to be sure em_javascript is loaded
```
Comment (olleharstedt) [inline scripts/numerical_input.js]: > do it at ready

It's not in ready?

**Comment by olleharstedt** [comment]
One fix should be one commit. Not four fixes in one. It's possible to have more than one commit in one pull request, right?

Which bug is this?

> Dev: Allow some fix/option in template

**Comment by Shnoulle** [comment]
thousand_epeartor are totally buggy, hard to fix only one part .....

For allow/fix options in template : https://github.com/LimeSurvey/LimeSurvey/pull/445/files#diff-e39de42c4425c990bedb3025ae48f9a4R21 is' a function, the a template function can replace it.

File: scripts/numerical_input.js (hunk 1 above)
Comment (Shnoulle) [inline scripts/numerical_input.js]: Yes, outdated comment ... sorry.

**Comment by Shnoulle** [comment]
Last git update fix the issue . @olleharstedt fix it

**Pull request closed without merging**